.travis.yml

```
@@ -1,10 +1,9 @@
 language: node_js
 node_js:
   - node
-  - 4
-  - 0.12
-  - "0.10"
-script: npm run cover
+  - 8
+  - 12
```

```suggestion
  - 10
```

`12` is covered by `node` 🙂

---

Oh ha my bad